apps/duellen/app/components/Start.jsx

```diff
@@ -58,32 +70,58 @@ export default class Start extends React.Component {
     this.props.history.push('/Intro');
   }
 
+  // Loads more quizzes
+  loadMore(e){
+    e.preventDefault();
+    try {
+      // Gets the more quizzes from the backend        
+      fetch(backendURL + 'get/all/quizzes/as/json/?lowerlim=' + this.state.lowerLim.toString() + '&upperlim='+ this.state.upperLim.toString())
+        .then((res) => {return res.json()})
+        .then((quizData) => {
+              this.setState({
+                quizData: [...this.state.quizData,  ...quizData],
+                lowerLim: this.state.lowerLim + this.state.gap,
+                upperLim: this.state.upperLim + this.state.gap,
+                showingAll: quizData.length < this.state.gap,
+              });
+            }
+          )
+     } catch (e) {
+       console.log(e);
+     }
+
+  }
 
   render() {
     try{
       return (
-        <div class="duellen--button-container" style={{marginTop: '-50px'}}>
+        <div className="duellen--button-container" style={{marginTop: '-50px'}}>
           <div className="mb-4 row justify-content-center">
             <div className='col-10 col-sm-10 col-md-7 col-lg-6'>
               <img src={Logo} alt="logo" style={{width: '100%', height:'100%'}}/>
             </div>
           </div>
           <MuiThemeProvider>
             <div>
-              <div class="text-center m-auto col-12">
+              <div className="text-center m-auto col-12">
                 <h3><b>{this.state.price}</b></h3>
                 <h2 className="header">Veckans Quiz</h2>
                 <a href={'/Intro/' + this.state.weeklyQuiz.id}>
-                  <Button variant="contained" class="start" fullWidth={true} color="primary">Vecka {this.getweek(this.state.weeklyQuiz.publication_date)}</Button>
+                  <Button variant="contained" className="start" fullWidth={true} color="primary">Vecka {this.getweek(this.state.weeklyQuiz.publication_date)}</Button>
                 </a>
                   <h2 className="slimh2">Tidigare Quiz</h2>
                 {this.state.quizData.map(item => (
-                  <div key={item.id} class="btnOrange">
+                  <div key={item.id} className="btnOrange">
                     <a href={'/Intro/' + item.id}>
-                      <Button variant="contained" class="start" color="primary" fullWidth={true}>Vecka {this.getweek(item.publication_date)}</Button>
+                      <Button variant="contained" className="start" color="primary" fullWidth={true}>Vecka {this.getweek(item.publication_date)}</Button>
                     </a>
                   </div>
                 ))}
+                {this.state.showingAll == false &&
```

Comment:
It would be better to have this as `this.state.showingAll === false && ...`.
Look https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a for context